telegrinder/tools/templating/jinja.py

```diff
@@ -0,0 +1,58 @@
+import pathlib
+import typing
+
+import jinja2
+
+from .abc import ABCTemplating
+
+
+def template_processing(template_source: str) -> str:
+    return "\n".join(map(str.strip, template_source.splitlines()))
```

Comment:
is it working correctly with lines consisting of jinja-expression? (they should be omitted)

Reply:
This is implemented for blocks via TAB. Example:

```jinja
{% for command in commands %}
    /{{ command.name }} - {{ command.description }}
{% endfor %}
```

done for the style of blocks so that the code does not seem like a mess without using TAB.